examples/passthroughfs.py

```diff
@@ -312,6 +313,14 @@ async def setattr(self, inode, attr, fields, fh, ctx):
             if fields.update_gid:
                 chown(path_or_fh, -1, attr.st_gid, follow_symlinks=False)
 
+            if fields.update_mode is False and \
+               fields.update_uid is False and \
+               fields.update_gid is False:
+                if chown == os.chown:
+                    chown(path_or_fh, -1, -1, follow_symlinks=False)
+                if chown == os.fchown:
+                    chown(path_or_fh, -1, -1)
```

Comment:
Hmm, this is rather strange:
- why is chown called at all if neither uid nor gid shall be set (both are -1)?
- is the `x is False` really wanted / correct instead of the usual boolean evaluation using `not x`? consider x == 0 or x == None, for example.
- in 311/314 `chown` is always called with `follow_symlinks=False`, no matter whether path or fh is used, so 320/322 look inconsistent with these.